### ISSUE TITLE
Relay client notified of own messages in direct message

### ISSUE THREAD

**Issue by mhoran** [wee-slack/wee-slack#594]
I use the [weechat-android](https://github.com/ubergeek42/weechat-android/) weechat relay client. Since the major refactor for wee-slack v2.0, I've begun receiving notifications of my own messages in direct messages.

Looking at `/debug tags`, the tags look pretty much identical:
```
11:34 adamkaplan │ ¯\_(ツ)_/¯ │ [nick_adamkaplan,slack_dm,irc_privmsg,notify_private,log1]
11:34     mhoran │ Haha.  [nick_mhoran,slack_dm,irc_privmsg,notify_private,log1]
```

I'm wondering if the `notify_private` on messages from myself is causing this? I'm not sure how that gets set.

I found a similar issue, #378, but none of the workarounds are applicable to relay clients.

Any help would be much appreciated!

**Comment by trygveaa** [comment]
This should have been fixed with cb5bfaf. Which version of wee-slack are you using?

**Comment by mhoran** [comment]
That did it! I thought I had upgraded to that but the timestamp on the file said otherwise. Thanks!
